src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -24,6 +24,8 @@
       RuntimeGraph="$(RuntimeIdentifierGraphPath)"
       NoWarn="$(NoWarn)"
       RunApiCompat="$([MSBuild]::ValueOrDefault('$(RunApiCompat)', 'true'))"
+      EnableStrictModeForCompatibleTfms="$([MSBuild]::ValueOrDefault('$(EnableStrictModeForCompatibleTfms)', 'true'))"
```

Comment:
Can you elaborate what's the difference between these two and why is on by default and the other is not?

Reply:
eg net6.0 assembly has some features which are not present in ns2.0, enabling the strict mode will mean that  everything in net6.0 needs to be in ns2.0 by default

Reply:
Right, but what behavior does compatible tfms and what does compatible frameworks cover? I feel like naming is confusing, they are very similar.

Reply:
CompatibleTargetFrameworks => covers cases that compileTime and runtimeTime (runtimeSpecific or runtime independent) are compatible with each other for any framework in the package.
CompatibleFrameworksInPackage => covers cases where a package has compatible frameworks inside the packge eg net5.0 & net6.0, ns2.0 and net5.0 and make sure that those assemblies satisfy the same compatibility rules.